rasa/core/policies/ted_policy.py

```diff
@@ -688,16 +688,20 @@ def train(
                 f"file to avoid this warning.",
                 category=UserWarning,
             )
-            return
+            return self._resource
 
         self.run_training(model_data, label_ids)
 
+        self.persist()
```

Comment:
💡 We always persist as part of the training

Reply:
Can we test for this?

Reply:
that's tested by the `PolicyTestCollection`. There are several tests which train and and then load from the persisted thing.